sdks/python/apache_beam/coders/coder_impl.pxd

```diff
@@ -137,3 +138,6 @@ cdef class WindowedValueCoderImpl(StreamCoderImpl):
 
   @cython.locals(c=CoderImpl)
   cpdef get_estimated_size_and_observables(self, value, bint nested=?)
+
+  @cython.locals(wv=windowed_value.WindowedValue)
+  cpdef encode_to_stream(self, value, OutputStream stream, bint nested)
```

Comment:
How about cython.returns decorator to declare the return type for decode_from_stream. Just thinking out loud, as this is new to me.